addon/components/ilios-calendar-event-month.js

```diff
@@ -11,13 +11,26 @@ export default CalendarEvent.extend({
   event: null,
   timeFormat: 'h:mma',
   classNameBindings: [':event', ':event-pos', ':ilios-calendar-event', 'event.eventClass', ':month-event'],
-  tooltipContent: computed('event', function(){
-    let str = this.get('event.location') + '<br />' +
-      moment(this.get('event.startDate')).format(this.get('timeFormat')) + ' - ' +
-      moment(this.get('event.endDate')).format(this.get('timeFormat')) + '<br />' +
-      this.get('event.name');
-  
-    return str;
+  tooltipContent: computed('event', function() {
+    const location = this.get('event.location');
+    const name = this.get('event.name');
+    const startTime = moment(this.get('event.startDate')).format(this.get('timeFormat'));
+    const endTime = moment(this.get('event.endDate')).format(this.get('timeFormat'));
+    const dueThisDay = this.get('dueThisDay');
+
+    let isILM = this.get('event.name').substring(0, 4) === 'ILM:';
```

Comment:
ILM events have an ilmSession property that is not null.  I don't think the name is standardized, but the ilmSession property is for sure going to work.